tensorflow/core/util/autotune_maps/conv_parameters.cc

```diff
@@ -94,7 +94,7 @@ MatmulParameters::MatmulParameters(
   proto_.set_c_dtype(c_dtype);
 
   proto_.set_trans_a(trans_a);
-  proto_.set_trans_b(trans_a);
+  proto_.set_trans_b(trans_b);
```

Comment:
This seems to be fixing a pre-existing bug that's not related to hipBlasLt? If so, is it better to put it in a separate commit?